rotkehlchen/tests/unit/test_general_cache.py

```diff
@@ -150,11 +174,11 @@ def mock_call_contract(contract, node_inquirer, method_name, **kwargs):
 
     # Check that the token was created
     token = GlobalDBHandler().get_evm_token(
-        address='0xD71eCFF9342A5Ced620049e616c5035F1dB98620',
```

Comment:
any reason why you changed the asset here?  DAI is always present in the DB iirc.